config.yml

```diff
@@ -71,3 +71,31 @@ languages:
         # - name: Boutique
         #   url: 'https://store.gobuffalo.io'
         #   weight: 6
+  es:
+    languageName: Spanish
+    languageCode: es-la
+    title: 🇪🇸 ES
+    weight: 1
```

Comment:
I think we need to change the weight value since the best practice is default language has the number 1 to make it possible to access them as `.Sites.First` (I already fixed that with my previous PR)

Reply:
Thanks!